src/client/app/components/Movie/MovieInfo.jsx

```diff
@@ -4,7 +4,7 @@ function MovieInfo (props) {
   var movie = props.movie;
   return (
     <div className="movie-info">
-      <div className="title">{movie.title}</div>
+      <div className="title" onClick={() => props.history.push('/film/'+movie.title)}>{movie.title}</div>
```

Comment:
It's better to move this function into a component's method with argument `title`